tools/run_tests/helper_scripts/build_csharp.sh

```diff
@@ -32,5 +32,12 @@ set -ex
 
 cd $(dirname $0)/../../../src/csharp
 
-# overriding NativeDependenciesConfigurationUnix is needed to make gcov code coverage work.
-xbuild /p:Configuration=$MSBUILD_CONFIG /p:NativeDependenciesConfigurationUnix=$CONFIG Grpc.sln
+OVERRIDE_NATIVELIB_MAYBE=""
+if [ "$CONFIG" == "gcov" ]
+then
+  # overriding NativeDependenciesConfigurationUnix makes C# project pick up
+  # the gcov flavor of grpc_csharp_ext
+  OVERRIDE_NATIVELIB_MAYBE="/p:NativeDependenciesConfigurationUnix=$CONFIG"
```

Comment:
NativeDependenciesConfigurationUnix=$CONFIG actually works fine for all configurations, but I'd like to test the default case where /p:NativeDependenciesConfigurationUnix is not set.
Your snippet is more readable, so I'll use it.